app/src/stores/settings.ts

```diff
@@ -17,7 +17,33 @@ export const useSettingsStore = defineStore({
 			const userStore = useUserStore();
 			if (!userStore.currentUser || 'share' in userStore.currentUser) return;
 
-			const response = await api.get(`/settings`);
+			const response = await api.get(`/settings`, {
+				params: {
+					fields: [
+						'project_name',
+						'project_url',
+						'project_color',
+						'project_logo',
+						'public_foreground',
+						'public_background',
+						'public_note',
+						'auth_login_attempts',
+						'auth_password_policy',
+						'storage_asset_transform',
+						'storage_asset_presets',
+						'custom_css',
+						'storage_default_folder',
+						'basemaps',
+						'mapbox_key',
+						'module_bar',
+						'project_descriptor',
+						'default_language',
+						'custom_aspect_ratios',
+						'image_editor',
```

Comment:
```suggestion
```

I believe this is only an alias field and doesn't need to be fetched 👍

https://github.com/directus/directus/blob/e5bf8a76f8368157bd10d609ab0b4417016ece2a/api/src/database/system-data/fields/settings.yaml#L383-L391